docs/docs/Concepts/lfx-headless-runner.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: Run flows with Langflow Executor (LFX)
+slug: /lfx-headless-runner
+---
+
+The Langflow Executor (LFX) is a command-line tool that serves and runs flows statelessly from [flow JSON files](/concepts-flows-import) with minimal dependencies.
+
+Flows are run without the flow builder UI or database, and any flow dependencies are automatically added to complete the run.
+The flow graph is stored in memory at all times, so there is less overhead for loading the graph from a database.
+
+LFX includes two commands for executing flows.
+
+* [`lfx serve`](#serve): This command starts a FastAPI server with your flow available at `/flows/{flow_id}/run`.
```

Comment:
Consider explicitly mentioning that it's a Langflow API endpoint hosted on a FastAPI server.

also consider linking to the reference page: https://docs.langflow.org/api-flows-run
(And, you may want to link from /api-flows-run to this page as well. Possibly as a "See also".)